### ISSUE TITLE
[Bug]: Unknown Hyprland errors pointing at /hypr/hyprland.conf

### ISSUE THREAD

**Issue by ayushvyas-1** [prasanthrangan/hyprdots#1005]
### Description

I updated my Arch (1/3/2024) and when rebooted i was shown three Hyprland errors.
### Steps to Reproduce

1. [Update Arch]

### Expected Behavior

Work Normally.

### Actual Behavior

[What actually happened?]

### Screenshots

![240302_11h31m08s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/77277708/64100192-2293-42ed-9818-49439b1cbc30)
But.
![240302_11h30m58s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/77277708/a72a527b-75c1-41d3-90d3-71fe058c6127)

### Environment

-  ArchLinux
- Linux A0 6.7.6-arch1-2 #1 SMP PREEMPT_DYNAMIC Thu, 29 Feb 2024 19:08:51 +0000 x86_64 GNU/Linux
- Hyprland, built from branch main at commit 555afea73c87f8e2bb374d1112431bb3a6f451dd dirty (makefile: add config to make asan).
Date: Fri Mar 1 19:46:56 2024
Tag: v0.36.0-21-g555afea7

- On The Main Hardware

**Comment by pinklifeart** [comment]
Hey, it's actually related to hyprscript update i believe - continious # is no longer a valid comment + they changed the device configuration syntax. You can solve it by adding whitespaces on lines 1 and 5 between the first # and the rest of them in hyprland.conf. 
```
######################################################################################
#AUTOGENERATED HYPR CONFIG.
#PLEASE USE THE CONFIG PROVIDED IN THE GIT REPO /examples/hypr.conf AND EDIT IT,
#OR EDIT THIS ONE ACCORDING TO THE WIKI INSTRUCTIONS.
########################################################################################
```
to
```
# #####################################################################################
#AUTOGENERATED HYPR CONFIG.
#PLEASE USE THE CONFIG PROVIDED IN THE GIT REPO /examples/hypr.conf AND EDIT IT,
#OR EDIT THIS ONE ACCORDING TO THE WIKI INSTRUCTIONS.
# #######################################################################################
```
As for the device config, you'd need to change it from 
```
device:epic mouse V1 {
    sensitivity = -0.5
}
```
to something like
```
device {
    name = epic mouse V1
    sensitivity = -0.5
}
```
[This commit might be helpful for you](https://github.com/prasanthrangan/hyprdots/pull/930/commits/dde434ac38df6bc71ea7fe177c68b9c562c498e4)

Also there seems to be a PR ready that covers this issue: #930

**Comment by Skerse** [comment]
Are you using the git version? Hyprland in non-testing Arch repos are still on 0.35.0 so if you use that version it's not broken. What's happening is Hyprland is migrating to Hyprlang and there are a few changes needed to fix the issues. Also yeah, https://github.com/prasanthrangan/hyprdots/pull/930 is going to be merged when 0.36.0 comes to main Arch repos as we follow that release, and if you want you can apply the changes in the pull request to hyprland.conf now.

**Comment by ayushvyas-1** [comment]
I changed my Hyprland-git (v0.36.0-21-g555afea7) to Hyprland (0.35.0) Arch repo's one and solved those errors,

This also solved issue in Hyprland-git. (https://github.com/prasanthrangan/hyprdots/issues/1005#issuecomment-1974694053)
Thanks.

(not related but waybar is also broken)
